src/commands/update-github-release.ts

```diff
@@ -12,10 +15,36 @@ type GitHubRepo = {
 
 export async function run(...args): Promise<boolean> {
   const argv = yargsParser(args);
+  const isDryRun = argv.dry;
+  let versionTag = argv.versionTag;
+  let title = argv.title;
+  let text = argv.text;
+
+  if (versionTag == null) {
```

Comment:
Same as above. 

Prefer:

```suggestion
  if (!versionTag) {
```

Otherwise, please use:

```suggestion
  if (versionTag === undefined) {
```

Reply:
s.o.